### PR TITLE
minor changes and signature

### PR DESCRIPTION
**Pull request by fippo**
see individual commit messages.

### CODE REVIEW

File: manifesto.txt [hunk 1]
```diff
@@ -23,7 +23,7 @@ o support the STARTTLS method in XMPP as specified in RFC 6120,
   validation consistent with RFC 6125
 
 o require the use of TLS for both client-to-server and 
-  server-to-server connections
+  server-to-server connections by default
```
Comment (imaginator) [inline manifesto.txt:26]: Good point. 

I'd go stronger and say "with no fallback to cleartext"

Comment (akuckartz) [inline manifesto.txt:26]: Simon Tennant:

> > @@ -23,7 +23,7 @@ o support the STARTTLS method in XMPP as specified in RFC 6120,
> >    validation consistent with RFC 6125
> > 
> >  o require the use of TLS for both client-to-server and 
> > -  server-to-server connections
> > -  server-to-server connections by default
> 
> Good point. 
> 
> I'd go stronger and say "with no fallback to cleartext"

+1

Cheers,
Andreas

Comment (fippo) [inline manifesto.txt:26]: As server developer I can hardly remove my users ability to shoot themselves in the foot. If they want unencrypted connection to google, then that is their choice. However, they should be absolutely aware that what they want is a bad idea.

Comment (stpeter) [inline manifesto.txt:26]: The idea here is that the operators who sign this statement will indeed prevent any fallback to cleartext. That might mean shutting off connectivity with Google Talk.

Comment (fippo) [inline manifesto.txt:26]: I do not have a problem with this at the operator level.
I do have a problem with this as developer of XMPP software, which are included in the preamble.

**Comment by stpeter** [comment]
IMHO, what software developers need to do is provide the configuration settings that enable a server operator to prevent fallback to cleartext. That's the intent here; if the intent isn't clear, we need to adjust the text.

**Pull request closed without merging**